src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/GitHookTasks.kt

```diff
@@ -33,13 +33,20 @@ open class InstallPrePushHookTask : InstallHookTask("pre-push") {
  * Install or update a kotlinter-gradle hook.
  */
 abstract class InstallHookTask(@get:Internal val hookFileName: String) : DefaultTask() {
-    @Input
+
+    @get:Input
```

Comment:
🔧 
If I'm not mistaken this doesn't matter for the generated bytecode, so I'd keep it as it was, just to keep it consistent with other tasks (i.e. `LintTask`) 
```suggestion
    @Input
```